.github/workflows/extra_tests.yml

```diff
@@ -301,6 +301,10 @@ jobs:
             libssl-dev \
             zlib1g
 
+      - name: Install obfs4proxy
+        run: |
+          apt-get install -y obfs4proxy
```

Comment:
Should probably also add something to the `docs/extra_tests.md` page about this dependency.